src/tokenlist.schema.json

```diff
@@ -241,7 +241,7 @@
         "symbol": {
```

Comment:
Should the pattern for `name` be updated as well?

Reply:
done

Reply:
done